ui/shared/components/table/DataTable.jsx

```diff
@@ -41,8 +41,15 @@ const StyledDataTable = styled(Table)`
 const ASCENDING = 'ascending'
 const DESCENDING = 'descending'
 
-const getRowColumnContent = (row, isExport) => (col, formatProps) => (
-  (col.format && !(isExport && col.noFormatExport)) ? col.format(row, isExport, formatProps) : row[col.name])
+const getRowColumnContent = (row, col, isExport, formatProps) => {
```

Comment:
do you remember why it was bound initially?

Reply:
I think because somebody though `columns.map(getRowColumnContent (row, true))` looked neater than `columns.map(col => getRowColumnContent (row, col, true))`